addons/account/models/product.py

```diff
@@ -63,24 +63,28 @@ def get_product_accounts(self, fiscal_pos=None):
     @api.depends('taxes_id', 'list_price')
     def _compute_tax_string(self):
         for record in self:
-            currency = record.currency_id
-            res = record.taxes_id.compute_all(record.list_price, product=record, partner=self.env['res.partner'])
-            joined = []
-            included = res['total_included']
-            if currency.compare_amounts(included, record.list_price):
-                joined.append(_('%s Incl. Taxes', format_amount(self.env, included, currency)))
-            excluded = res['total_excluded']
-            if currency.compare_amounts(excluded, record.list_price):
-                joined.append(_('%s Excl. Taxes', format_amount(self.env, excluded, currency)))
-            if joined:
-                record.tax_string = f"(= {', '.join(joined)})"
-            else:
-                record.tax_string = " "
-
+            record.tax_string = self._construct_tax_string(record, record.list_price)
+    @api.model
+    def _construct_tax_string(record, price):
+        currency = record.currency_id
+        res = record.taxes_id.compute_all(price, product=record, partner=record.env['res.partner'])
+        joined = []
+        included = res['total_included']
+        if currency.compare_amounts(included, price):
+            joined.append(_('%s Incl. Taxes', format_amount(record.env, included, currency)))
+        excluded = res['total_excluded']
+        if currency.compare_amounts(excluded, price):
+            joined.append(_('%s Excl. Taxes', format_amount(record.env, excluded, currency)))
+        if joined:
+            tax_string = f"(= {', '.join(joined)})"
+        else:
+            tax_string = " "
+        return tax_string
 
 class ProductProduct(models.Model):
     _inherit = "product.product"
 
+    tax_string = fields.Char(compute='_compute_tax_string')
```

Comment:
add an empty line after
Is it even needed? I don't see anything that is changed from the inherited field declaration. Redefining the computed function implementation should be enough.

Reply:
@william-andre 
I tried that but it didn't work, the product.template implementation is one that is always called. I though it is because of the way that [delegation inheritance (https://www.odoo.com/documentation/15.0/developer/reference/backend/orm.html#delegation) works

Reply:
Anyways, add an empty line, and adapt the dependencies.

I'll test it afterwards :)